### ISSUE TITLE
Public assemblies should have a strong name

### ISSUE THREAD

**Issue by tom-englert** [serilog-contrib/serilog-sinks-grafana-loki#215]
### Which version of Serilog.Sinks.Grafana.Loki are you using?

8.1

### Which version of .NET are you using?

net48

### Describe the bug

It's not possible to use the sink with a NetFramework project that has a strong name

### To Reproduce

Try to use the sink in a NetFramework project with a strong name:

At compile time:
```
CSC : warning CS8002: Referenced assembly 'Serilog.Sinks.Grafana.Loki, Version=8.1.0.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.
```

At runtime:
```
System.IO.FileLoadException
Could not load file or assembly 'Serilog.Sinks.Grafana.Loki, Version=8.1.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
```

### Expected behavior

We can compile and run NetFramework projects with a strong name, using this sink.

### Log/SelfLog output or exception with stacktrace

System.IO.FileLoadException
Could not load file or assembly 'Serilog.Sinks.Grafana.Loki, Version=8.1.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)

### I have read the documentation

- [X] I have read [Common Problems](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems) section in project wiki

**Comment by mishamyte** [comment]
Hi @tom-englert!

I believe we had a small discussion in #155

**Comment by mishamyte** [comment]
Hi! 

Implemented in [v8.2.0-beta.3](https://www.nuget.org/packages/Serilog.Sinks.Grafana.Loki/8.2.0-beta.3). Could you pls validate it?

Plan to release `v8.2.0` on Monday if all would be good

**Comment by tom-englert** [comment]
8.2.0 works as expected 👍
